index.js

```diff
@@ -200,6 +202,10 @@ class Ky {
 
 		const headers = new Headers(this._options.headers || {});
 
+		if (((supportsFormData && this._options.body instanceof FormData) || this._options.body instanceof URLSearchParams) && headers.has('content-type')) {
+			throw new Error('The `content-type` header cannot be used with a FormData body. It will be set automatically.');
```

Comment:
Could maybe use `this._options.body.name` instead of hardcoding `FormData` in the error message.

Reply:
done